exercicios-para-casa/ex07.js

```diff
@@ -1,13 +1,21 @@
 // Estrutura for
 
 // Faça um algoritmo que mostre os números de 0 até 100 
-[Substitua este texto por sua resposta]
+[let inicio = 0
```

Comment:
Aqui faltou escrever o for
ex: for (i=inicio; i<=fim;i+=passo){
console.log(i)
}